content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+
+[`Mat3A`]: https://docs.rs/bevy/0.8.0/bevy/math/struct.Mat3A.html
+[`Vec3`]: https://docs.rs/bevy/0.8.0/bevy/math/struct.Vec3.html
+[`Vec3A`]: https://docs.rs/bevy/0.8.0/bevy/math/struct.Vec3A.html
+[`Quat`]: https://docs.rs/bevy/0.8.0/bevy/math/struct.Quat.html
+
+## AABB Recalculation
+
+<div class="release-feature-authors">authors: @mlodato517</div>
+
+When a [`Mesh`] asset is changed, the AABB (axis aligned bounding box) is now recalculated, ensuring frustum culling calculations are still correct.
+
+## SpatialBundle and VisibilityBundle
+
+<div class="release-feature-authors">authors: @mockersf, @rparrett</div>
+
+With the addition of [Visibility Inheritance](#visibility-inheritance) comes the constraint that visibility propagation requires all elements in the hierarchy to have the appropriate visibility components. When constructing scenes, developers often want to group entities under parent "organizational" entities, which exist solely to group entities together, reposition them, and hide them as a unit. These "organizational" entities still require visibility components to propagate the [`Transform`] and [`Visibility`] to [`GlobalTransform`] and [`ComputedVisibility`] (respectively).
+
+To make this easy, we've added a new [`SpatialBundle`], which adds the components mentioned above. This allows the entity to configure and propagate visibility and transform data without incurring the cost of actually rendering it.
+
+```rust
+commands
+    // This entity controls the position and visibility of the entities beneath it.
+    .spawn_bundle(SpatialBundle {
+        transform: Transform::from_xyz(10.0, 20.0, 30.0),
+        visibility: Visibility {
+            is_visible: true,
+        },
+        ..default()
+    }).with_children(|parent| {
+        parent
+            .spawn_bundle(TableBundle::default())
+            .spawn_bundle(ShopKeeperBundle::default())
+            .spawn_bundle(PotionBundle::default());
+    });
+```
+
+If you know you don't need [`Transform`] propagation (or your entity already has those components), you can instead use the new [`VisibilityBundle`], which only adds the components required for visibility propagation.
+
+[`Transform`]: https://docs.rs/bevy/0.8.0/bevy/transform/struct.Transform.html
+[`GlobalTransform`]: https://docs.rs/bevy/0.8.0/bevy/transform/struct.GlobalTransform.html
+[`SpatialBundle`]: https://docs.rs/bevy/0.8.0/bevy/render/struct.SpatialBundle.html
+[`VisibilityBundle`]: https://docs.rs/bevy/0.8.0/bevy/render/view/struct.VisibilityBundle.html
+
+## ShaderType derive
+
+<div class="release-feature-authors">authors: @teoxoy</div>
+
+**Bevy 0.8** now uses the [`ShaderType`] trait / derive (provided by the [encase][encase] crate) to easily convert Rust data types to GPU-compatible shader data types. 
```

Comment:
`[encase][encase]` looks like a typo?
```suggestion
**Bevy 0.8** now uses the [`ShaderType`] trait / derive (provided by the [encase](https://crates.io/crates/encase) crate) to easily convert Rust data types to GPU-compatible shader data types. 
```

Reply:
Nope! Its just using a different link format that lets us define it in a footer. Just scroll down a bit!

Reply:
Ah all good then. Not used to such syntax :)